test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright 2023 Arm Limited and/or its affiliates <open-source-office@arm.com>
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+SSHHOST="${SSHHOST:?SSHHOST is unset (or empty)}"
```

Comment:
Er, no, and there probably should be. It needs to be some Morello board (or model), like our other examples, though I omitted the `SSHPORT` logic that we've used before because it's often easier to define `~/.ssh/config` aliases for them anyway (and I tested this on a real host).

I should document this.